lib/rucio/tests/test_transfer.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Radu Carpa <radu.carpa@cern.ch>, 2021
+
+import unittest
+
+import pytest
+
+from lib.rucio.core.transfer import get_hops
+from rucio.common.config import config_get, config_get_bool
+from rucio.common.exception import NoDistance
+from rucio.common.types import InternalAccount
+from rucio.core.importer import import_data
+from rucio.core.rse import get_rse_id, del_rse
+from rucio.tests.common import rse_name_generator
+
+
+class TestTransfer(unittest.TestCase):
```

Comment:
just a side-note: do you know about pytest fixtures? https://docs.pytest.org/en/stable/fixture.html
I try to gradually replace `unittest.TestCase`, because we can save a lot of test code and therefore time with fixtures.

Reply:
I already used them in the past. Now that I know it's OK to use them in rucio, I'll definitely use them more often.